handbook/engineering/developer-insights/index.md

```diff
@@ -100,6 +100,27 @@ In addition to team specific channels, we communicate in the following org-wide
 
 [#developer-insights-org-leadership](https://sourcegraph.slack.com/archives/C01P1TVDJR4): This channel is for the Engineering Managers, Product Managers and Product Designers to discuss higher-level matters impacting the whole org.
 
+#### Acknowledgment
+
+In an async-first communication environment it is important to remove assumptions/uncertainty around whether teammates have seen, understood or acted on a message.  
+To assist in this regard we provide the following guideline for teammates to follow when communicating and responsing in Slack. 
+
+**The most important thing to remember is not which emoji to use, but rather to remember to acknowledge and to do it in a clear and unambiguous way.**
+
+*When acknowledging a request:*
+
+- `:thumbsup:` (👍) = acknowledgment as in I see the request and will action it
```

Comment:
I usually use 👍🏻  to endorse something, but not necessarily act on it myself (e.g. maybe it was a request to someone else, or a group of people I am a part of but someone else owns it). This description here would kind of make me worried I can't do that anymore as it may get misunderstood (and it's a really common emoji reaction). Can we resolve that?

Reply:
Thinking about this...

If responding 👍 to a request addressed to someone else (either directly or a group you are no part of) I think it would be fine.

If responding 👍 to a request addressed to a group of people you are a part of but you have no intention of actioning it, I would question whether it's the best reaction to use? Maybe there's something more appropriate to use to voice your support like the "+1" emoji or :point_up:?

Do you maybe have some ideas on how we could change this to be better. Maybe use the :ack: emoji reaction?

Reply:
(I agree with Jean for both cases here – if it's like "Hey code insights team can you all email me your favorite snack" and you thumbs up it, I would assume you were going to act on that.)